src/mobile/apps/fair_organizer/routes.coffee

```diff
@@ -42,6 +42,7 @@ module.exports.fetchFairOrgData = (req, res, next) ->
   request = fairs.fetch
     data:
       fair_organizer_id: fairOrg.id
+      sort: "-start_at"
```

Comment:
👍to do it on the API level.

Reply:
One thing I would note is that from the comment above (on line 40), it seems to be a product decision to fetch "past" fairs (and also the articles from those past fairs on the same page.) I am not very familiar with the fairs page on mobile, so I would suggest reaching out to the Fairs team and @joanna20, and talk about the changes here (i.e. now we will fetch all the fairs by the same organizer sorted by starting date (and also their articles).

If people feel good about this change, let's also modify the comment.